R/rnet_junctions.R

```diff
@@ -0,0 +1,30 @@
+#' Get points where 3 or more linestrings meet (junctions on road networks)
+#'
+#' @inheritParams rnet_breakup_vertices
+#' @export
+#' @examples
+#' has_sfheaders <- requireNamespace("sfheaders", quietly = TRUE)
+#' if(has_sfheaders) {
+#' rnet <- rnet_roundabout
+#' j <- rnet_junction(rnet)
+#' plot(rnet$geometry)
+#' plot(j, add = TRUE)
+#' }
+rnet_junction <- function(rnet) {
+  stopifnot(requireNamespace("sfheaders", quietly = TRUE))
+  coordinates <- sfheaders::sf_to_df(rnet)
+  # names(coordinates)
+  # head(coordinates)
+  coordinates <- as.matrix(coordinates[c("x", "y", "linestring_id")])
```

Comment:
I'm sorry but I don't understand why you convert the data frame to a matrix

Reply:
`sf` required a matrix. But `sfheaders` does not (I think) - will update.